src/CommunityToolkit.Aspire.Hosting.RavenDB/RavenDBBuilderExtensions.cs

```diff
@@ -123,16 +123,19 @@ private static Dictionary<string, object> GetEnvironmentVariablesFromServerSetti
         {
             environmentVariables.TryAdd("RAVEN_PublicServerUrl", securedServerSettings.PublicServerUrl);
             environmentVariables.TryAdd("RAVEN_Security_Certificate_Path", securedServerSettings.CertificatePath);
-            
+
             if (securedServerSettings.CertificatePassword is not null)
                 environmentVariables.TryAdd("RAVEN_Security_Certificate_Password", securedServerSettings.CertificatePassword);
         }
 
         return environmentVariables;
     }
 
-    private static void ConfigureEnvironmentVariables(EnvironmentCallbackContext context, Dictionary<string, object>? environmentVariables = null)
+    private static void ConfigureEnvironmentVariables(EnvironmentCallbackContext context, RavenDBServerResource serverResource, Dictionary<string, object>? environmentVariables = null)
     {
+        context.EnvironmentVariables.TryAdd("RAVEN_ServerUrl_Tcp", () => $"{serverResource.TcpEndpoint.Scheme}://0.0.0.0:{serverResource.TcpEndpoint.Port}");
+        context.EnvironmentVariables.TryAdd("RAVEN_PublicServerUrl_Tcp", () => serverResource.TcpEndpoint.Url);
```

Comment:
```suggestion
        context.EnvironmentVariables.TryAdd("RAVEN_ServerUrl_Tcp", $"{serverResource.TcpEndpoint.Scheme}://0.0.0.0:{serverResource.TcpEndpoint.Port}");
        context.EnvironmentVariables.TryAdd("RAVEN_PublicServerUrl_Tcp", serverResource.TcpEndpoint.Url);
```

I don't think you want to use a callback here, as it's resulting in https://github.com/CommunityToolkit/Aspire/actions/runs/13850817873/job/38757679398?pr=538#step:6:177

This change should fix it.

Reply:
You're right, I wrongly copied that from another place that accepts a callback.

Reply:
All good, the number of "whoops" commits I have here...🤣